content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -91,241 +91,81 @@ Depending on the version of Istio, you may see destination rules for hosts other
-sleep.foo to httpbin.foo: 503
-sleep.foo to httpbin.bar: 503
-sleep.bar to httpbin.foo: 503
-sleep.bar to httpbin.bar: 503
-{{< /text >}}
-
-To configure the client side, you need to set [destination rules](/docs/concepts/traffic-management/#destination-rules) to use mutual TLS. It's possible to use
-multiple destination rules, one for each applicable service (or namespace). However, it's more convenient to use a rule with the `*` wildcard to match all
-services so that it is on par with the mesh-wide authentication policy.
-
-{{< text bash >}}
-$ kubectl apply -f - <<EOF
-apiVersion: "networking.istio.io/v1alpha3"
-kind: "DestinationRule"
-metadata:
-  name: "default"
-  namespace: "istio-system"
-spec:
-  host: "*.local"
-  trafficPolicy:
-    tls:
-      mode: ISTIO_MUTUAL
-EOF
-{{< /text >}}
-
-{{< tip >}}
-* Starting with Istio 1.1, only destination rules in the client namespace, server namespace and `global` namespace (default is `istio-system`) will be considered for a service, in that order.
-* Host value `*.local` to limit matches only to services in cluster, as opposed to external services. Also note, there is no restriction on the name or
-namespace for destination rule.
-* With `ISTIO_MUTUAL` TLS mode, Istio will set the path for key and certificates (e.g client certificate, private key and CA certificates) according to
-its internal implementation.
-{{< /tip >}}
+This policy specifies that all workloads in the mesh will only accept encrypted requests using TLS. As you can see, this authentication policy contains no `selector` specification (as it is intended to apply to all services in the mesh).
 
-Don’t forget that destination rules are also used for non-auth reasons such as setting up canarying, but the same order of precedence applies. So if a service
-requires a specific destination rule for any reason - for example, for a configuration load balancer -  the rule must contain a similar TLS block with
-`ISTIO_MUTUAL` mode, as otherwise it will override the mesh- or namespace-wide TLS settings and disable TLS.
+This policy directly instructs Pilot to program sidecar proxy to accept only mutual TLS inbound traffic. With `auto mTLS` feature enabled (see more [here](/docs/tasks/security/auto-mtls)), it will also try to configure client to use mTLS when possible. Also note, the authentication policy doesn't have effect on workload that doesn't have sidecar (e.g `httpbin.legacy`). However, `auto mTLS` also takes that into account and does not apply mTLS to the outbound traffic to the legacy service.
 
-Re-running the testing command as above, you will see all requests between Istio-services are now completed successfully:
+If you run the test command above, you will see all requests still succeed, except for those from client that doesn't have sidecar (`sleep.legacy`) to server with sidecar (`httpbin.foo` or `httpbin.bar`). This is expected, as the mTLS is now strictly required, but the workload without sidecar cannot comply.
```

Comment:
What test command? Please repeat the command if other things are in between.
```suggestion
Run the test command again:
`<insert test command>`
You see requests still succeed, except for those from the client that doesn't have proxy, `sleep.legacy`, to the server with a proxy, `httpbin.foo` or `httpbin.bar`. This is expected because mTLS is now strictly required, but the workload without sidecar cannot comply.
```

Reply:
done